build/webpack/webpack.extension.web.config.js

```diff
@@ -32,23 +32,6 @@ const config = {
     },
     module: {
         rules: [
-            {
-                // JupyterServices imports node-fetch.
```

Comment:
As soon as I started importing these into the web bundle, this code failed to work.

I believe we don't need in the webbundle anymore.

Reply:
Yeah, I'm not seeing the node-fetch usage anymore in jupyterlab services.